src/Equinox.MessageDb/MessageDb.fs

```diff
@@ -48,6 +49,8 @@ module Log =
         | Some retryPolicy ->
             let withLoggingContextWrapping count =
                 let log = if count = 1 then log else log |> prop contextLabel count
+                let act = Activity.Current
+                if act <> null then act.AddTag("eqx.attempts", count) |> ignore
```

Comment:
push out to `Tracing.AddRetryAttempt` or something so it becomes clear if we add top level atttempt for OCC loop

Maybe internal labels should be `eqx.sync_attempt`/`transact_attempt` for top level and  `eqx.op_attempt` for low level retries?